submissions/krylenger/dom-api-task/script.js

```diff
@@ -0,0 +1,183 @@
+      "They are popular for green laning on public byways / un-tarmacked roads.",
+    ],
+    models: ["HONDA CRF450RX", "HUSQVARNA FX450", "KTM 450XC-F"],
+    avgPrice: "Average price: $11.000",
+  },
+  {
+    name: "RETRO",
+    img: "./images/retro.png",
+    description: [
+      "Most of the retro models you can buy today started life as a sporty naked bike which has been modified to look more traditional, but without sacrificing all the modern tech and riding experience you would expect from a new bike.",
+      "They usually feature a single round headlight, simpler rounded clocks, stitched seats, more subtle colour schemes, rounded exhausts, loads of optional shiny bits and often wire spoked wheels or cast wheels designed to look a little bit like spokes from a distance.",
+      "Closely related are modern cafe racers which tend to start life as retro naked bikes but with the addition of low bars, a headlight fairing and pillion seat hump.",
+    ],
+    models: ["Kawasaki Z900RS", "Triumph Speed Twin", "Moto Guzzi V7 III"],
+    avgPrice: "Average price: $10.000",
+  },
+];
+
+const navBtnToggle = document.querySelector(".nav-btn-toggle");
+const nav = document.querySelector(".nav");
+const navUl = document.querySelector(".nav__ul");
+const main = document.querySelector(".main");
+const mainHeader = document.querySelector(".main__header");
+const mainImage = document.querySelector(".main__img");
+const mainDescription = document.querySelector(".main__description");
+const mainDescriptionText = document.querySelector(".main__description-text");
+const mainUlBikes = document.querySelector(".main__ul-bikes");
+const mainAvgPrice = document.querySelector(".main__avg-price");
+
+
+const handleNavBtnToggle = ({ target }) => {
+
+    navBtnToggle.classList.toggle("nav-btn-toggle--nav-closed");
+    nav.classList.toggle("nav--closed");
+    main.classList.toggle("main--wide");
+    navUl.classList.toggle("nav__ul--display-none");
+};
+
+const handleNavigationClick = ({ target }) => {
+  if (target.nodeName === 'A') {
+    const sectionName = target.textContent;
+    const section = database.find((item) => item.name.toLowerCase() === sectionName.toLowerCase());
+    mainHeader.textContent = section.name;
+    mainImage.setAttribute("src", section.img);
+    mainAvgPrice.textContent = section.avgPrice;
+  
+    const fragmentBikes = document.createDocumentFragment();
+    const fragmentDescription = document.createDocumentFragment();
+  
+    section.models.forEach((model) => {
+      const bike = document.createElement("li");
+      bike.textContent = model;
+      fragmentBikes.appendChild(bike);
+    });
+    
+    section.description.forEach((description) => {
+      const desc = document.createElement("p");
+      desc.textContent = description;
+      desc.classList.add("main__description-p");
+      fragmentDescription.appendChild(desc);
+    });
+    
+    const navLinks = document.querySelectorAll(".nav__link");
```

Comment:
No need to query all elements with `nav__link--orange` className because you have only one this element. You can try this approach. To my mind, it's simpler (only 2 lines of code)
``` javascript
const activeElement = document.querySelector('.nav__link.nav__link--orange');
activeElement.classList.remove("nav__link--orange");
```

Reply:
Didn't get it.
I need navLinks with all nav__link (s) inside because I'm looping through all links.
This I need to add orange color to active link and remove orange from the previous active link.
Without loop I don't know how implement this.

Reply:
My suggested approach will help you to find a link with the orange color. After that you can remove `nav__link--orange` className from this element and add this class to the `target` element. Let me know if it still isn't clear

Reply:
Get it, 
✅ Done